CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Version 13.3.0 - 2024-11-29
+
+- Bugfix `transitions update-execution` now supports additional keyword arguments
+- Added optional parameter `--run-async` to `predictions create`
```

Comment:
Should this parameter better reflect the API or the Python SDK? I think API is the correct answer

```suggestion
- Added optional parameter `--async` to `predictions create`
```